.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Can you remove it from the commit and add to the .gitignore at the end .vscode/ 🙏

Reply:
I think I did it...